opentelemetry-api/src/opentelemetry/propagators/__init__.py

```diff
@@ -104,9 +118,25 @@ def inject(
     get_global_textmap().inject(set_in_carrier, carrier, context)
 
 
-_HTTP_TEXT_FORMAT = composite.CompositeHTTPPropagator(
-    [TraceContextTextMapPropagator(), BaggagePropagator()],
-)  # type: textmap.TextMapPropagator
+try:
+
+    propagators = []
+
+    for propagator in (  # type: ignore
+        Configuration().get("PROPAGATORS", "tracecontext,baggage").split(",")  # type: ignore
+    ):
+
+        propagators.append(  # type: ignore
+            next(  # type: ignore
+                iter_entry_points("opentelemetry_propagator", propagator)  # type: ignore
```

Comment:
Do we need to add entry points for `"tracecontext` and `baggage` as well?

Reply:
They're already defined here no? 
https://github.com/open-telemetry/opentelemetry-python/pull/1123/files#diff-0440b84b95b1038a8ed782c70cc345efR59-R60

Reply:
What...must be tripping.